ignite/pkg/protopath/protopath.go

```diff
@@ -73,3 +75,15 @@ func ResolveDependencyPaths(ctx context.Context, src string, versions []module.V
 
 	return paths, nil
 }
+
+// FormatPackageName formats a protocol buffer package name from a module path.
+func FormatPackageName(ownerName, appName, moduleName string) string {
```

Comment:
This looks like hacky? Do we need this func, after fixing the frontend code gen?

Reply:
A function to create proto package names is required because we have to deal with the 2 and 3 level names ("app.module" or "namespace.app.module"). Escaping is also required for the package name prefix ("app" or "namespace") because the "namespace" might come from a GitHub username which might contain dashes or start with a number.

In https://github.com/ignite-hq/cli/pull/2353/commits/df62ae356054f1d5f853e00c5f94862b43f6e5e3 the `FormatPackageName` was moved and renamed to `ignite.templates.module.ProtoPackageName`:
```go
ProtoPackageName(appModulePath, moduleName string) string
```
This function is now used to generate the value for `protoPkgName` that is used in the proto file templates.